### PR TITLE
Loss update to multilevel for pfam and go 

### PR DESCRIPTION
**Pull request by DrHB**

### CODE REVIEW

**Comment by hypnopump** [comment]
Seems you're replacing  a multiclass-CrossEntropy with a binary one: https://github.com/hypnopump/rgn2-replica/pull/12/files#diff-b9c79841c1d8de5ba8801237dbe57579e39613126e063a4583a464231161278cR48

 Is this correct?

**Comment by DrHB** [comment]
correct me if I am wrong, my understanding is that  we should use `nn.BCEWithLogitsLoss` because it  treats each output independently and is suitable for multi-label classification ? (because `pfam` and `go` are mulit-label`)

**Comment by hypnopump** [comment]
Oh okay, i thought we were doing softmax at last layer, my bad. Okay then, i'll merge!
